<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Numpad</class>
 <widget class="QWidget" name="Numpad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumbpad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="sieben">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>221</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acht">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>221</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="neun">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>221</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vier">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>304</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="funf">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>304</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sechs">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>304</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eins">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>387</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zwei">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>387</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="drei">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>387</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delte">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zero">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enter">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>470</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>⎆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zforSpin">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yforSpin">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="xforSpin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="geschw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Geschwindigkeit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
